cns/NetworkContainerContract.go

```diff
@@ -17,11 +17,12 @@ const (
 	SetOrchestratorType                      = "/network/setorchestratortype"
 	CreateOrUpdateNetworkContainer           = "/network/createorupdatenetworkcontainer"
 	DeleteNetworkContainer                   = "/network/deletenetworkcontainer"
-	GetNetworkContainerStatus                = "/network/getnetworkcontainerstatus"
 	PublishNetworkContainer                  = "/network/publishnetworkcontainer"
 	UnpublishNetworkContainer                = "/network/unpublishnetworkcontainer"
 	GetInterfaceForContainer                 = "/network/getinterfaceforcontainer"
 	GetNetworkContainerByOrchestratorContext = "/network/getnetworkcontainerbyorchestratorcontext"
+	GetAllNetworkContainers                  = "/network/networkcontainers"
+	NetworkContainerRefreshAssociation       = "/network/networkcontainerrefreshassociation"
```

Comment:
Saw another comment to this effect which I agree with: a more RESTful approach is to think of these as operations on a `networkcontainers` collection:
- GET `/network/networkcontainers` returns all the NCs in CNS's state file
- PUT `/network/networkcontainers` allows client to send a request body with the NCs that need to be stored in CNS's state file. (Could also be a POST IMO since there's no standard for bulk request as far as I'm aware of).

This would be a bit more declarative if CNS used an http routing library, but in this case it's a bit barebones, so to be clear, I'm suggesting:
- A single route, no `/network/networkcontainerrefreshassociation`
- A single handler for the route, which dispatches to appropriate methods:
```go
func (s *HTTPRestService) networkContainers(w http.ResponseWriter, r *http.Request){
    switch r.Method {
    case http.MethodGet:
        getNetworkContainers(w,r)
    case http.MethodGet:
        putNetworkContainers(w,r)
    default:
        w.WriteHeader(http.StatusMethodNotAllowed)
    }
}
```

Reply:
Merged into one method, and use a single route.
If received "GET":
1. Check NCs in CNS's state file, return "false" if any NC is missing. It means NC refresh association is needed;
2. If all NCs exist, return all NCs in CNS's state file for follow-up check;

If received "POST": Store all the NCs (from the request body that client sent) into CNS's state file